js/modules/k6/http/http_test.go

```diff
@@ -46,6 +79,7 @@ func TestTagURL(t *testing.T) {
 	for expr, data := range testdata {
 		expr, data := expr, data
 		t.Run("expr="+expr, func(t *testing.T) {
+			t.Parallel()
```

Comment:
How is this not panicking when it's using the same runtime for all 5 cases?

Reply:
Weird, I don't know, but I'll fix it

Reply:
Resolved by the [latest force push](https://github.com/grafana/k6/compare/f072e4d180fb41c3686dd0c44a6ae8fbd0783193..18919f3f0f95645ace7c0a7ba1f71423978509e8)